nvflare/lighter/impl/master_template.yml

```diff
@@ -366,6 +366,8 @@ sub_start_svr_sh: |
   DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
   echo "WORKSPACE set to $DIR/.."
   mkdir -p $DIR/../transfer 
+  export PYTHONPATH=/local/custom:$PYTHONPATH
+  echo "PYTHONPATH is $PYTHONPATH"
```

Comment:
Actually, this is to fix this line new_env["PYTHONPATH"] = new_env["PYTHONPATH"] + ":" + app_custom_folder.  I believe a better way to fix it is new_env["PYTHONPATH"] = new_env.get("PYTHONPATH", "") + ":" + app_custom_folder

Reply:
let me make that changes thanks